virttest/test_setup.py

```diff
@@ -2029,16 +2030,54 @@ def stop(self):
         self._compress_log()
 
 
-def disable_smt():
+def disable_smt(rmt=False, params=None):
     """
     Checks whether smt is on, if so disables it in PowerPC system
-    This function is used in env_process to check & disable smt in Power8
+    This function is used in env_process and in libvirt.py to check
+    & disable smt in Power8 for local and remote machine respectively.
+
+    :param rmt: If True, turns off smt on remote machine
```

Comment:
No need 'rmt', just deal with it by checking required parameters(server/remote_ip ...)